### PR TITLE
Lean on the remote carpentries-theme

### PR DESCRIPTION
**Pull request by tkphd**
rather than hauling around the assets & boilerplate ourselves. This improves portability & maintainability by streamlining the repository: almost all our content, minimal upkeep otherwise.

### CODE REVIEW

**Review by ocaisa — CHANGES_REQUESTED** [review]
This looks good to me, there are a few complaints from the lesson check to resolve and we can drop the travis file now we have GitHub actions

File: bin/lesson_initialize.py [hunk 1]
```diff
@@ -47,5 +47,11 @@ def main():
         )
 
 
+if __name__ == '__main__':
```
Comment (ocaisa) [inline bin/lesson_initialize.py]: This whole section doesn't look right, I think there's a copy/paste error here

**Review by ocaisa — CHANGES_REQUESTED** [review]

**Comment by tkphd** [comment]
Fixt. Thanks @ocaisa!

File: bin/lesson_initialize.py (hunk 1 above)
Comment (ocaisa) [reply bin/lesson_initialize.py]: Same issue resolved for hpc-intro resolved in https://github.com/hpc-carpentry/hpc-intro/pull/207

Comment (ocaisa) [reply bin/lesson_initialize.py]: Fixed it here so this is good to go

**Review by ocaisa — APPROVED** [review]
Couple of warnings about long lines but they can be fixed in a separate PR

**Pull request merged**